onnxruntime/core/providers/webgpu/shader_variable.cc

```diff
@@ -55,7 +60,11 @@ constexpr static const std::string_view VALUE_TYPE_ARRAY[] = {
     "u32",         // Uint8x4 (u32 as 4 elements of uint8)
     "vec2<u32>",   // Uint8x8 (vec2<u32> as 2x4 elements of uint8)
     "vec4<u32>",   // Uint8x16 (vec4<u32> as 4x4 elements of uint8)
-    "i32",         // Int8x4
+    "u32",         // Int8x4 (u32 as 4 elements of uint8)
+    "vec2<i32>",   // Int8x8 (vec2<i32> as 2x4 elements of int8)
```

Comment:
is this expected to be u32 instead of i32? if storage type is u32 but value type is i32, there need explicit bitcast

Reply:
made it i32